KclosestElementsTwoPointers.py

```diff
@@ -0,0 +1,33 @@
+import heapq as heap
+'''Time Complexity: O(n-k)
```

Comment:
Log(N) + k will be the TC. Check the lecture if you have confusion regarding TC for both the problems